Server/src/main/java/org/bsu/twiter/forms/GetObjectFromForm.java

```diff
@@ -0,0 +1,38 @@
+package org.bsu.twiter.forms;
+
+import org.bsu.twiter.dao.TwitDAOImpl;
+
+import java.lang.reflect.Field;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+public class GetObjectFromForm {
+
+    private static Logger logger;
+
+    static {
+        try {
+            LogManager.getLogManager().readConfiguration(GetObjectFromForm.class.getClassLoader().getResourceAsStream("logger.properties"));
+            logger = Logger.getLogger(TwitDAOImpl.class.getName());
+        } catch (Exception ignored) {
+        }
+    }
+
+    public static <T> T getObject(Object form, Class<T> objectClass) {
```

Comment:
Using reflection is cool... But to be honest, looks like a great deal over-engineering here!))
Techically you are only calling for this method to populate a `Twit` from `TwitCreateForm`. 
And even if you had `Twit` and `Twit2`, it would be more understandable to have just two overloads of `getObject()` for both.
And even if there were `Twit`, `Twit2` and `Twit3`, you **still** wouldn't need such a generified solution.
What are the bottlenecks here:
1) it is *strictly implied* that both `form` and `objectClass` have equally named fields
2) it is implied that *all* fields of `form` *must* be "visible" for `objectClass`
So technically you are performing *object cloning* of `form` into `objectClass`
But why then not using `form` object itself? There are exra questins arising out of this solution.

But still reflection is truly a thing to master)